### ISSUE TITLE
Color codes still appear in files, when you try to output the script to a file via terminal

### ISSUE THREAD

**Issue by seb26** [seb26/mhl-compare#2]
`mhl-compare any.mhl file.mhl` > `test.log`

produces

```
1st MHL file: [32m/Users/seb/Development/mhl-compare/samples_mhl/2019-03-17_2019-03-17_212416.mhl[0m
              [32m270 files[0m
              [32m5.3 GiB (5682964630 bytes)[0m
```

Instead of:

```
1st MHL file: /Users/seb/Development/mhl-compare/samples_mhl/2019-03-17_2019-03-17_212416.mhl
              270 files
              5.3 GiB (5682964630 bytes)
```

**Comment by seb26** [comment]
Potential solution(s):
https://stackoverflow.com/questions/46971566/python-to-handle-color-to-terminal-and-also-in-file

1. Create a custom color function to wrap `termcolor`'s function
2. Inside the function, check if `os.isatty(1)` is true or false
3. Only output termcolor's color if `os.isatty(1)` is true

**Comment by seb26** [comment]
Resolved in b64fa85
